### ISSUE TITLE
Toggle visibility two times to hide when --show --new-tab=/

### ISSUE THREAD

**Issue by AhmedKamal20** [Guake/guake#1443]
- **Guake Version**: 0.8.4
  - GNU/Linux Ubuntu 16.04 64bit Unity

When i run this command : 
`guake --show --new-tab=/`
I have to press F12 (Toggle visibility) two times before guake actually hide

**Comment by gsemet** [comment]
Hello. Guake 0.8 is no more supported. Please update to 3 through pip install

**Comment by AhmedKamal20** [comment]
a i had some problems when installing it using pip.
is this fixed in v3 ?

**Comment by gsemet** [comment]
mostly, if any issue are left, please tell me

**Comment by Akito13** [comment]
> mostly, if any issue are left, please tell me

I have the same issue on
```
Guake Terminal: 3.4.0
VTE: 0.54.2
VTE runtime: 0.54.2
Gtk: 3.24.1
```

When running without extra arguments on Debian Buster, KDE Plasma, SDDM.

**Comment by david-kimpel** [comment]
I am on

```
Guake Terminal: 3.3.0
VTE: 0.54.1
VTE runtime: 0.54.1
Gtk: 3.24.1
```

And I always have to toggle the visibility twice for Guake to actually hide. It will show up after 1 toggle though.

**Comment by aichingm** [comment]
@Akito13 @david-kimpel do you have refocus-if-open enabled?

**Comment by Akito13** [comment]
> @Akito13 @david-kimpel do you have refocus-if-open enabled?

So I turned it off and the hiding works now as it should. Thank you!

**Comment by david-kimpel** [comment]
@aichingm Thanks for the suggestion, that worked!

**Comment by mlouielu** [comment]
Duplicate of #1499, should be fixed by #1529, tested on master `HEAD` will show up the terminal and hide by one toggle.
